CMakeLists.txt

```diff
@@ -34,7 +34,6 @@ include(GNUInstallDirs)
 
 option(MRDOX_INSTALL "Configure install target" ON)
 option(MRDOX_PACKAGE "Build install package" ON)
-option(MRDOX_BUILD_SHARED "Link shared" ${BUILD_SHARED_LIBS})
```

Comment:
If we're really messing seriously with shared libraries, should we be testing this in CI?

Reply:
For instance,

```
          factors: |
            gcc Shared
            msvc Shared
# ...
    uses: ... cmake-workflow
        # ...
        with:
            # ... 
           shared: ${{ matrix.shared }}
```

or a union of whatever factors are already there...

Here's an example: https://github.com/alandefreitas/cpp-actions/blob/develop/.github/workflows/ci.yml

Reply:
It is tested through the test-plugin.

Reply:
Oh... Sorry. The MRDOX_BUILD_SHARED support is actually being removed. OK.